R/ggplotly.R

```diff
@@ -1377,6 +1377,10 @@ ggtype <- function(x, y = "geom") {
   sub(y, "", tolower(class(x[[y]])[1]))
 }
 
+get_first <- function(x){
+  if(length(x)) x[[1]] else x
+}
```

Comment:
This is not a useful abstraction

Reply:
using `[[1]]` directly resulted in errors when `gglayout$yaxis$ticktext|tickvals` was empty and instead of making the code incomprehensible by cramming more conditions I decided on adding a simple function that checked whether x is empty or not

Reply:
I get what it's doing, but it's not worth making a function out of it (especially now that #2062 is merged)